docs/semgrep-secrets/policies.md

```diff
@@ -0,0 +1,120 @@
+---
+slug: policies
+append_help_link: true
+title: Policies
```

Comment:
Can we change this to a short phrase that tells the user what they do with this thing called a "policy"?